lib/Service/AiIntegrations/AiIntegrationsService.php

```diff
@@ -50,6 +50,8 @@ class AiIntegrationsService {
 
 	/** @var IMailManager  */
 	private IMailManager $mailManager;
+	public const SummaryTask = SummaryTaskType::class;
```

Comment:
what is the idea behind these constants?

conventional php consts are all upper case

Reply:
In order to use `isLlmAvailable()` for different task types.

Reply:
But you could also call isLlmAvailable with SummaryTaskType::class directly

Reply:
I just didn't want to add extra imports in other files, but I can change it

Reply:
I believe the code is easier to read without the const/alias